LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (C) 2015 President and Fellows of Harvard College
+Copyright (C) 2016 Python LTI Initiative
```

Comment:
Hm, not sure what the best way forward is. I'm not affiliated with Top Hat these days, and maybe @aehlke might have some thoughts.

I'm not sure anybody from Top Hat is really on top of LTI stuff anymore.